hal_api-rails.gemspec

```diff
@@ -1,36 +1,35 @@
-# coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'hal_api/rails/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "hal_api-rails"
+  spec.name          = 'hal_api-rails'
   spec.version       = HalApi::Rails::VERSION
-  spec.authors       = ["Chris Rhoden", "Andrew Kuklewicz"]
-  spec.email         = ["carhoden@gmail.com", "andrew@beginsinwonder.com"]
+  spec.authors       = ['Chris Rhoden', 'Andrew Kuklewicz']
+  spec.email         = ['carhoden@gmail.com', 'andrew@beginsinwonder.com']
 
-  spec.summary       = %q{JSON HAL APIs on Rails in the style of PRX}
-  spec.description   = %q{JSON HAL APIs on Rails in the style of PRX v4. Uses ROAR}
-  spec.homepage      = "https://www.github.com/PRX/"
-  spec.license       = "MIT"
+  spec.summary       = 'JSON HAL APIs on Rails in the style of PRX'
+  spec.description   = 'JSON HAL APIs on Rails in the style of PRX v4. Uses ROAR'
+  spec.homepage      = 'https://www.github.com/PRX/'
+  spec.license       = 'MIT'
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
+  spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  spec.require_paths = ['lib']
 
-  spec.add_dependency "activemodel", ">= 3.0.0"
-  spec.add_dependency "actionpack", ">= 3.0.0"
-  spec.add_dependency "rack-test"
-  spec.add_dependency "activesupport", ">= 3.0.0"
-  spec.add_dependency "responders", "~> 3.0"
-  spec.add_dependency "roar-rails", "~> 1.1.0"
-  spec.add_dependency "multi_json"
+  spec.add_dependency 'activemodel'
+  spec.add_dependency 'activesupport'
+  spec.add_dependency 'multi_json'
+  spec.add_dependency 'roar-rails'
 
-  spec.add_development_dependency "bundler"
-  spec.add_development_dependency "rake", "~> 10.0"
-  spec.add_development_dependency "rails", "~> 5.2.4"
-  spec.add_development_dependency "pry-byebug"
-  spec.add_development_dependency "minitest"
-  spec.add_development_dependency "kaminari"
+  spec.add_development_dependency 'bundler'
+  spec.add_development_dependency 'kaminari'
+  spec.add_development_dependency 'minitest'
+  spec.add_development_dependency 'pry-byebug'
+  spec.add_development_dependency 'rack-test'
+  spec.add_development_dependency 'rails', '~> 5.2.4'
```

Comment:
6 might be more accurate these days.  But this is probably fine - minimize changes!